iphone/Classes/TiAppiOSUserActivityProxy.h

```diff
@@ -16,6 +16,13 @@
 - (id)initWithOptions:(NSDictionary *)props;
 @property (nonatomic, strong) NSUserActivity *userActivity;
 
+#if IS_XCODE_10
+- (void)setEligibleForPrediction:(NSNumber *)value;
+- (void)setPersistentIdentifier:(NSString *)value;
+- (void)deleteSavedUserActivitiesForPersistentIdentifiers:(id)persistentIdentifiers;
+- (void)deleteAllSavedUserActivities:(id)unused;
```

Comment:
Not only the new methods but all methods ;-). Including code-docs and params-descriptions.

Reply:
All method exposing is fine. Do we use code-docs and params-description in header file in Titanium SDK?

Reply:
For Ti core-developers, it will help knowing which params are used in the "boxed" arguments. But it looks fine for now.